From 48aced9a72f0a9aedc98630e3b8e18452f923ff9 Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@freescale.com>
Date: Fri, 8 Jun 2012 07:48:54 -0500
Subject: [PATCH 287/502] ENGR00211339: TalkingTomCat: Tom Cat cannot talk.

Talking TomCat does not get the mic input since the Mic
Gain is not set as per linux-3.0.15 kernel driver.

Signed-off-by: Nitin Garg <nitin.garg@freescale.com>
---
 alsa/alsa_imx.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/alsa/alsa_imx.cpp b/alsa/alsa_imx.cpp
index b4273e2..4869ca1 100755
--- a/alsa/alsa_imx.cpp
+++ b/alsa/alsa_imx.cpp
@@ -690,7 +690,7 @@ void setDefaultControls(uint32_t devices, int mode, const char *cardname)
         if(selecteddevice == DEVICE_SGTL5000)
         {
             if(devices & AudioSystem::DEVICE_IN_BUILTIN_MIC){
-                ctl->set("MIC GAIN","20dB");
+                ctl->set("Mic Volume", 2, 0);
             }
         }
     }
-- 
1.8.0

